EasyPost.Tests.Net/EventTest.cs

```diff
@@ -7,29 +8,67 @@ namespace EasyPost.Tests.Net
     public class EventTest
     {
         [TestInitialize]
-        public void Initialize() => ClientManager.SetCurrent("NvBX2hFF44SVvTPtYjF0zQ");
+        public void Initialize()
+        {
+            VCR.SetUp(VCRApiKey.Test, "event", true);
+        }
+
+        private static EventCollection GetBasicEventCollection()
+        {
+            return Event.All(new Dictionary<string, object>
+            {
+                {
+                    "page_size", Fixture.PageSize
+                }
+            });
+        }
 
         [TestMethod]
-        public void TestLoad() => Assert.AreEqual(Resource.Load<Event>("{'id': 'barfoo'}").id, "barfoo");
+        public void TestAll()
+        {
+            VCR.Replay("all");
+
+            EventCollection eventCollection = GetBasicEventCollection();
+
+            List<Event> events = eventCollection.events;
+
+            Assert.IsTrue(events.Count <= Fixture.PageSize);
+            Assert.IsNotNull(eventCollection.has_more);
+            foreach (var item in events)
+            {
+                Assert.IsInstanceOfType(item, typeof(Event));
+            }
+        }
 
         [TestMethod]
-        [ExpectedException(typeof(HttpException))]
         public void TestRetrieve()
         {
-            // Events are archived after some time. Lets at least make sure we get a 404.
-            Event e = Event.Retrieve("evt_d0000c1a9c6c4614949af6931ea9fac8");
+            VCR.Replay("retrieve");
+
+            // use "TestAll"
+            EventCollection eventCollection = GetBasicEventCollection();
+            Event _event = eventCollection.events[0];
```

Comment:
why is this variable begins with an underscore?

Reply:
`event` is a reserved word in C#, so had to name the temporary variable something else.